yt_dlp/extractor/orf.py

```diff
@@ -583,31 +584,65 @@ class ORFONIE(InfoExtractor):
             'upload_date': '20240128',
         }
+    }, {
+        'url': 'https://on.orf.at/video/3220355',
+        'md5': 'f94d98e667cf9a3851317efb4e136662',
+        'info_dict': {
+            'id': '3220355',
+            'ext': 'mp4',
+            'duration': 445.04,
+            'thumbnail': 'https://api-tvthek.orf.at/assets/segments/0002/60/thumb_159573_segments_highlight_teaser.png',
+            'title': '50 Jahre Burgenland: Der Festumzug',
+            'description': 'md5:1560bf855119544ee8c4fa5376a2a6b0',
+            'media_type': 'episode',
+            'timestamp': 52916400,
+            'upload_date': '19710905',
+        }
     }]
 
-    def _extract_video(self, video_id, display_id):
+    def _extract_video(self, video_id):
         encrypted_id = base64.b64encode(f'3dSlfek03nsLKdj4Jsd{video_id}'.encode()).decode()
         api_json = self._download_json(
-            f'https://api-tvthek.orf.at/api/v4.3/public/episode/encrypted/{encrypted_id}', display_id)
+            f'https://api-tvthek.orf.at/api/v4.3/public/episode/encrypted/{encrypted_id}', video_id)
+
+        if api_json.get('is_drm_protected'):
+            self.report_drm(video_id)
 
         formats, subtitles = [], {}
         for manifest_type in traverse_obj(api_json, ('sources', {dict.keys}, ...)):
             for manifest_url in traverse_obj(api_json, ('sources', manifest_type, ..., 'src', {url_or_none})):
                 if manifest_type == 'hls':
                     fmts, subs = self._extract_m3u8_formats_and_subtitles(
-                        manifest_url, display_id, fatal=False, m3u8_id='hls')
+                        manifest_url, video_id, fatal=False, m3u8_id='hls')
                 elif manifest_type == 'dash':
                     fmts, subs = self._extract_mpd_formats_and_subtitles(
-                        manifest_url, display_id, fatal=False, mpd_id='dash')
+                        manifest_url, video_id, fatal=False, mpd_id='dash')
                 else:
                     continue
                 formats.extend(fmts)
                 self._merge_subtitles(subs, target=subtitles)
 
+        for subtitle_type in ['vtt']:  # not working formats 'xml', 'srt', 'sami', 'ttml', 'stl'
+            subtitle_url = traverse_obj(api_json, ('_embedded', 'subtitle', f'{subtitle_type}_url'), {str})
+            if subtitle_url is None:
+                continue
+            self._merge_subtitles({
+                'de': [
+                    {
+                        'url': subtitle_url,
+                        'ext': f'{subtitle_type}',
+                    }
+                ],
+            }, target=subtitles)
+
+        age_classification = traverse_obj(api_json, ('age_classification'), {str})
+        age_limit = parse_age_limit(age_classification)
+
         return {
             'id': video_id,
             'formats': formats,
             'subtitles': subtitles,
+            'age_limit': age_limit,
             **traverse_obj(api_json, {
```

Comment:
You can just inline this:
```suggestion
            **traverse_obj(api_json, {
                'age_limit': ('age_classification', {parse_age_limit}),
```

Reply:
not so familiar with `traverse_obj` magic ;)
thanks.

Reply:
ah one moment fix my last commit